src/installer/installer.go

```diff
@@ -669,6 +685,25 @@ func (i *installer) waitForBootkube(ctx context.Context) {
 	}
 }
 
+func (i *installer) waitForNodeImageOverlay(ctx context.Context) {
```

Comment:
Probably it can be change to just call utils.WaitForPredicate with 
i.ops.SystemctlAction("is-active", "node-image-overlay.service")
WDYT?

Reply:
What happens if the overlay service fails? Should this be able to detect that and return an error?